applications/callflow/src/cf_endpoint.erl

```diff
@@ -661,6 +662,18 @@ maybe_do_not_disturb(Endpoint, _, _) ->
             {'error', 'do_not_disturb'}
     end.
 
+-spec maybe_exclude_from_queues(wh_json:object(), wh_json:object(), whapps_call:call()) ->
+                                        'ok' |
+                                        {'error', 'exclude_from_queues'}.
+maybe_exclude_from_queues(Endpoint, _, Call) ->
+    QueueId = whapps_call:custom_channel_var(<<"Queue-ID">>, Call),
```

Comment:
Can we do something to make this short-circuit faster?

```erlang
case is_binary(whapps_call:custom_channel_var(<<"Queue-ID">>, Call))
    andalso wh_json:is_true(<<"exclude_from_queues">>, Endpoint)
of
    'false' -> 'ok';
    'true' -> {'error', 'excluded'}
end.
```

Something like this, to me, reads better as to the intention of the code (versus mashing the two toggles into a tuple).

Thoughts?